### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**
Hi, here is my homework.
FCC user name: rguinsberg

### CODE REVIEW

File: javascript/javascript1/week1/js-week1.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+// Smart-ease
+// Age-ify (A future age calculator)
+const yearOfBirth = 1975;
+const yearFuture = 2045;
+
+let age = yearFuture - yearOfBirth;
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: why let?

File: javascript/javascript1/week1/js-week1.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+// console.log(age);
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: please remove unused part of the code

File: javascript/javascript1/week1/js-week1.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+console.log("You will be " + age + " " + "years old in " + yearFuture + ".");
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: javascript/javascript1/week1/js-week1.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+// Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2035;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: why let?

File: javascript/javascript1/week1/js-week1.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    console.log("Your dog will be " + dogYear + " " + "dog years old in " + dogYearFuture + ".");
+}
+else {
+    console.log("Your dog will be " + dogYear + " " + "human years old in " + dogYearFuture + ".")
+}
+
+
+
+// House Pricey (A house price calculator)
+const peterHouseWidth = 8;
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: Imagine you can put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8]` `const gardenSizeInM2 = [100,70]`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?

File: javascript/javascript1/week1/js-week1.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+// Ez Namey (Startup name generator) 
+const firstWords = ["Easy", "Awesome", "Corporate", "Start", "The Best", "Future", "Great", "Tech", "Professional", "Fractal"];
+const secondWords = [" For you", " Solutions", " Now", " Innovations", " Choice", " Way", " Data", " Treasure", " Pixel", " Digital"];
+
+const randomNumber = Math.floor(Math.random() * 10);
+
+let startupName = firstWords[randomNumber] + secondWords[randomNumber];
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: why let?

File: javascript/javascript1/week1/js-week1.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    console.log("Julia is paying too much for the new house!")
+}
+
+
+
+// Ez Namey (Startup name generator) 
+const firstWords = ["Easy", "Awesome", "Corporate", "Start", "The Best", "Future", "Great", "Tech", "Professional", "Fractal"];
+const secondWords = [" For you", " Solutions", " Now", " Innovations", " Choice", " Way", " Data", " Treasure", " Pixel", " Digital"];
+
+const randomNumber = Math.floor(Math.random() * 10);
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: What if you need two random numbers per array? 🙂

File: javascript/javascript1/week1/js-week1.js [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+if (shouldShowResultInDogYears) {
+    dogYear = dogYear * 7;
+    console.log("Your dog will be " + dogYear + " " + "dog years old in " + dogYearFuture + ".");
+}
+else {
+    console.log("Your dog will be " + dogYear + " " + "human years old in " + dogYearFuture + ".")
+}
+
+
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js:23]: Prettier 🙂

File: javascript/javascript1/week1/js-week1.js [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+const juliaHouseWidth = 5;
+const juliaHouseDepth = 11;
+const juliaHouseHight = 8;
+const juliaHouseGardenSizeInM2 = 70;
+
+const juliaHouseSellingPrice = 1000000;
+
+let juliaHouseVolumeInMeters = juliaHouseWidth * juliaHouseDepth * juliaHouseHight;
```
Comment (cecastosic) [inline javascript/javascript1/week1/js-week1.js]: Try to read more about where to use let and where const. Maybe this video can help https://youtu.be/1mgLWu69ijU

**Review by cecastosic — COMMENTED** [review]
Great job @RodrigoGuinsbergPinto Please check my comments to make some improvements 🌸

File: javascript/javascript1/week1/js-week1.js (hunk 3 above)
Comment (RodrigoGuinsbergPinto) [reply javascript/javascript1/week1/js-week1.js]: More elegant and easier!
Thanks.

File: javascript/javascript1/week1/js-week1.js (hunk 7 above)
Comment (RodrigoGuinsbergPinto) [reply javascript/javascript1/week1/js-week1.js]: I could not figure out what you mean by two numbers per array. Can you give me a hint?

Comment (cecastosic) [reply javascript/javascript1/week1/js-week1.js]:
```
const firstRandomNumber = Math.floor(Math.random() * 10);
const secondRandomNumber = Math.floor(Math.random() * 10);
const startupName = firstWords[firstRandomNumber] + secondWords[secondRandomNumber];
```

**Comment by RodrigoGuinsbergPinto** [comment]
Thank you. I have implemented my homework as you have suggested.

**Pull request merged**